pyxlsb/styles.py

```diff
@@ -1,9 +1,50 @@
 import os
 from . import recordtypes as rt
 from .recordreader import RecordReader
+from .records import FormatRecord
 
 
 class Styles(object):
+
+    _builtin_formats = {
+        0: FormatRecord(fmtId=0, fmtCode='General'),
```

Comment:
One more thing, if you have some kind of source with these definitions, can you add the link or reference in a comment?